### ISSUE TITLE
Missing grouping treatments on parent containers of inputs

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2427]
Containers of related form controls are largely missing grouping treatments that expose the relationship of the inputs to assistive tech. This is mostly the case within forms; notably, in instances where `.inputs` divs effectively group a set of inputs, the appropriate roles are missing. Another instance is within `.sorting` forms in browse views. See the [WAI tutorial on Grouping Controls](https://www.w3.org/WAI/tutorials/forms/grouping/) for more.
